.github/workflows/ci.yml

```diff
@@ -33,6 +33,8 @@ jobs:
       run: xvfb-run --auto-servernum yarn test-bundle
     - name: yarn test-docs
       run: xvfb-run --auto-servernum yarn test-docs
+    - name: yarn test-viewer
+      run: xvfb-run --auto-servernum yarn test-viewer
```

Comment:
needs `yarn build-all` (to get `yarn build-viewer`), `xvfb`, and is quite fast (11s locally, we'll see in CI), so here in `basics` seemed like the obvious choice.

Reply:
> 11s locally, we'll see in CI

13s